modules/snooker/src/main/ts/ephox/snooker/resize/Bar.ts

```diff
@@ -3,11 +3,15 @@ import { Attr, Css, Element } from '@ephox/sugar';
 const col = function (column: number, x: number, y: number, w: number, h: number) {
   const blocker = Element.fromTag('div');
   Css.setAll(blocker, {
-    position: 'absolute',
-    left: x - w / 2 + 'px',
-    top: y + 'px',
-    height: h + 'px',
-    width: w + 'px'
+    'position': 'absolute',
+    'left': x - w / 2 + 'px',
+    'top': y + 'px',
+    'height': h + 'px',
+    'width': w + 'px',
+    '-webkit-user-select': 'none',
+    '-moz-user-select': 'none',
+    '-ms-user-select': 'none',
+    'user-select': 'none'
```

Comment:
Weren't these `user-select` variants in the CSS style for the blocker class? If not, would that be a better place for them? 🤔

Reply:
you're quite right, there's a content style sheet for snooker that I could put this in...

also this was misnamed at some point as these are the bars themselves and not blockers, I'll open another pr

Reply:
excellent!